.travis.yml

```diff
@@ -7,8 +7,8 @@ go:
 
 before_install:
   # Setup dependency management tool
-  - curl -L -s https://github.com/golang/dep/releases/download/v0.4.1/dep-linux-amd64 -o $GOPATH/bin/dep
-  - chmod +x $GOPATH/bin/dep
+  - curl https://nixos.org/nix/install | sh && . ~/.nix-profile/etc/profile.d/nix.sh
+  - nix-env -i dep
```

Comment:
I'm curious to learn why it's better to use `nix-env` (and introducing another dependency) vs grabbing from github releases: do you think the release management of nix is more secure?

Reply:
As far as I know Terraform will provide fetching providers as modules. There are ongoing works about this.